Desktop development

inakiesk opened the pull request:
This branch is dedicated to creating a desktop version of the Portfolio and it includes the following:
-HTML basic structure of desktop version.
-CSS styling for desktop version.
-Adaptable positioning for desktop version.
-Media queries setup for the different elements between mobile and desktop versions.
-Correct image sizes for desktop version.

tresorsawasawa commented on index.html:
```diff
@@ -19,14 +20,20 @@
     <div class="logo">
       <p><a href="./index.html">My Logo</a></p>
     </div>
```
- [ ] Kindly make sure your logo text(**"My Logo"**) follows the styling of your chosen Figma template to stick to the Figma design. e.g.: text-decoration, text color once clicked,...

- [ **OPTIONAL** ] For good tags semantics I suggest you remove the `<p>` tag to make your code base looks more professional i.e it can only look like this: 
```
   <div class="logo">
      <a href="./index.html">My Logo</a>
   </div>
```

tresorsawasawa commented on style.css:
```diff
@@ -1,595 +1,601 @@
-  background-color: black;
-  margin-top: 32px;
-  margin-left: 96px;
+@media (max-width: 480px) {
+  .desktop {
```
- [ ] According to the project requirements, I just noticed that the media queries for the Desktop breakpoint in the CSS file are supposed to have the value for the `min-width` property to be in the range of `768px `or `wider`) and not `480px`. Make sure you fix this to match the project requirements.

tresorsawasawa commented on style.css:
```diff
@@ -1,595 +1,601 @@
-body {
-  display: flex;
-  align-items: center;
-  flex-direction: column;
-}
-
-.container {
-  display: flex;
-  box-sizing: border-box;
-  height: 652px;
-  width: 375px;
-  border-style: solid;
-  border-width: 0;
-  border-color: black;
-  flex-direction: column;
-}
-
-.toolbar {
-  display: flex;
-  height: 48px;
-  width: 375px;
-  justify-content: space-between;
-}
-
-.logo {
-  margin-left: 12px;
-  margin-top: 4px;
-  margin-bottom: 4px;
-  padding: 10px 12px;
-  width: 100px;
-  height: 40px;
-  display: flex;
```
- [ ] **IMPORTANT:** As I mentioned in previous comments, kindly make sure your `Mobile` version still working as expected and not removing its styling, then the styling in the `@media` query will work when you are in a Desktop screen width size.
Example of one behavior of your project: **Header:** **`On mobile`** you will see the `Humbarger/menu` button but not the navigation options(Portfolio, About, Contact) but on **`On Desktop`** it will be the opposite(`Humbarger/menu` button will be hidden and display the navigation options)

tresorsawasawa reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @inakiesk 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

wuletawwonte commented on style.css:
```diff
@@ -1,595 +1,602 @@
+    display: flex;
+    box-sizing: border-box;
+    height: 652px;
+    width: 375px;
```
- [ ] As commented by the previous reviewer make sure the mobile version still looks like the design. You did good bringing back the code but the content of **the mobile version should fill the entire width of the viewport**. I think it would be nice to remove the white space on both sides of the website. you can achieve that by using `%` or `vw` to set the width of your containers.

![imageedit_1_9067656841](https://user-images.githubusercontent.com/12524453/178045508-26f75d26-ed06-4dae-97ff-048e22140603.jpg)

wuletawwonte reviewed (changes requested):
# Hi @inakiesk 👋🏿👋🏿👋🏿  

Good job so far!
There are some issues that you still need to work on to go to the next project! You have only a few more steps and you are done! 💪 💪 💪 

![giphy](https://user-images.githubusercontent.com/74506933/153382064-ed760a8c-4f4a-48d6-b1f0-1aa9e5135097.gif)

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Nice commit history ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

gtekle commented on index.html:
```diff
@@ -19,14 +20,20 @@
+    <nav class="menu-icon-container mobile">
       <img src="images/toolbar-icon.png" alt="Menu icon">
     </nav>
+    <ul class="toolbar-list desktop">
+      <li>Portfolio</li>
+      <li>About</li>
+      <li>Contact</li>
+    </ul>
```
- [ Optional] Even if this is optional it is highly recommended that you make the menu items links as suggested by the previous reviewer.
```
    <ul class="toolbar-list desktop">
      <li><a href="#">Portfolio</a></li>
      ...
    </ul>
```

gtekle commented on index.html:
```diff
@@ -126,6 +132,96 @@ <h2 class="connect">LET’S CONNECT</h2>
 
 </section>
 
+<section class="works-desktop desktop">
+  <article class="work1">
+    <img src="images/work1-desktop-img.svg" alt="Work image">
+    <div class="work-text-container">
+      <header class="work-header">Tonic</header>
+      <ul class="work-subheader">
+        <li class="list1">CANOPY</li>
+        <li class="dot"></li>
+        <li class="list2">Back End Dev</li>
+        <li class="dot"></li>
+        <li class="list2">2015</li>
+      </ul>
+      <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```
- [ ] You should not duplicate your HTML code for the sole purpose of rendering it in different screen sizes. That is the job of CSS not HTML. Thus, kindly remove the duplicated code for desktop and have only one WORKS section HTML code and style it using CSS media query so that the layout will adapt to different screen sizes.
![image](https://user-images.githubusercontent.com/44057863/178073566-64609a0d-ea64-4b6b-a8ba-d59393a9e6cf.png)

- [ Optional ] The background color for the `works` section is same as the `body` which is `#e5e5e5`.

inakiesk replied:
@gtekle It would be nice if all of these changes were brought to me on my first code review instead of having to spend all my reviews doing the changes suggested only to get different things to implement every time. I would be more than glad to push them all in a single review! Just a thought.

gtekle replied:
I completely agree with you 💯 and I apologize for the inconvenience. 
You are making good progress in addressing all the requested changes, though. Keep pressing forward and happy coding 👍

inakiesk replied:
No need to apologize at all! I find your feedback very helpful, that also motivated me to clean all my CSS code while I was at it. It took a while but my CSS looks decent now! Good practice always pays off!

gtekle reviewed (changes requested):
Hi @inakiesk :wave: , 

# STATUS: CHANGES REQUESTED :red_circle: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![almost-there-01](https://user-images.githubusercontent.com/44057863/178073863-706a23a9-596f-4a2e-867b-b8acecd2ffa4.gif)

## Highlights
:heavy_check_mark: Professional Readme
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Nice UI design

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly check the inline comment below

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

inakiesk commented:
Summary of new corrections:
-Removed duplicated HTML code and used CSS media query for better display as requested.
-Added links to menu items in toolbar for faster navigation as suggested.
-Changed background color in works section as suggested, looks better.
-Reorganized all CSS and media query code for better display and understanding, removed several duplicate codes while keeping original website design.
![](https://media.giphy.com/media/xT9IgCMZB3QCQYwwIo/giphy.gif)

danielufeli reviewed (approved):
Hi @inakiesk ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![a](https://camo.githubusercontent.com/40c24648ada128030359495b9504e95985194c3f4bbeabf0485e740ecd253bcc/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6a4a5143327075565a70544d4f34765573302f67697068792e676966)

## Highlights
- No Linter Errors✔️
- Clean Code ✔️
- Clean Design ✔️
- Descriptive Readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sinansevgi reviewed (commented):
Hello @inakiesk 👋,

_This is an additional review as a result of the second opinion request._

I can see that your project is already approved. I was happy to see that the problem has been resolved by itself and that you have completed your project.

Keep rocking 🚀! 
Sinan Sevgi
